cmd/cloudx/client/handler.go

```diff
@@ -153,6 +153,19 @@ func NewCommandHelper(cmd *cobra.Command) (*CommandHelper, error) {
 	}, nil
 }
 
+func (h *CommandHelper) GetDefaultProjectID() (string, error) {
+	conf, err := h.readConfig()
+	if err != nil {
+		return "", err
+	}
+
+	if conf.SelectedProject != uuid.Nil {
+		return conf.SelectedProject.String(), nil
+	}
+
+	return "", errors.New("No valid default project was specified")
```

Comment:
Go errors are lower case :)

```suggestion
	return "", errors.New("no valid default project was specified")
```